src/TitleDetails/AddPieceModal/AddPieceModal.js

```diff
@@ -19,7 +19,7 @@ import {
 import stripesFinalForm from '@folio/stripes/final-form';
 import {
   FieldDatepickerFinal,
-  FieldLocationFinal,
+  FieldLocation,
```

Comment:
why we need this change?

Reply:
I've updated component name cause we need to use it for react-final-form and redux form (in Orders)

Reply:
Is it possible to keep the name, so it's a less change? We have this pattern already, when several components are exported, like FieldSelect and FieldSelectFinal, etc.

Reply:
I've updated common component in [stripes-acq-components](https://github.com/folio-org/stripes-acq-components/pull/283), so this PR could be closed.